src/Caliburn.Micro.Avalonia/Platforms/netcore-avalonia/BootstrapperBase.cs

```diff
@@ -159,6 +157,24 @@ protected virtual void BuildUp(object instance)
         {
         }
 
+        /// <summary>
+        /// Override this to add custom behavior to execute after the application starts.
+        /// </summary>
+        /// <param name="sender">The sender.</param>
+        /// <param name="e">The args.</param>
+        protected virtual void OnStartup(object sender, ControlledApplicationLifetimeStartupEventArgs e)
+        {
+        }
+
+        /// <summary>
+        /// Override this to add custom behavior on exit.
+        /// </summary>
+        /// <param name="sender">The sender.</param>
+        /// <param name="e">The event args.</param>
+        protected virtual void OnExit(object sender, ControlledApplicationLifetimeExitEventArgs e)
```

Comment:
In the other BootstrapperBase this is EventArgs instead of ExitEventArgs, but for OnStartUp it uses the more specific args. Was there a reason to use EventArgs for exit or is it ok to have ControlledApplicationLifetimeExitEventArgs like this?